lib/util.js

```diff
@@ -60,7 +60,8 @@ const getSingleListDataFromSingleURL = (listURL, filter) => {
         body = filter(body)
       }
       body = sanitizeABPInput(body)
```

Comment:
`sanitizeABPInput` was originally meant to be for including all things that modify ABP input so you could not worry about where it comes from.  I think it'd be good to add `filterRareRulesPromise` inside there. You could convert it to a promise as well if you want. You'd just add another filter predicate there and fetch the list on startup of the tool.